hetmech/bulk_computation.py

```diff
@@ -0,0 +1,58 @@
+import pandas
+import scipy.special
+import scipy.stats
+
+import hetmech.degree_group
+import hetmech.degree_weight
+import hetmech.hetmat
+
+
+def compute_save_dgp(hetmat, metapath, damping=0.5, compression='gzip', delete_intermediates=True):
+    """
+    Compute summary file of combined degree-grouped permutations (DGP). Aggregates across permutations,
+    deleting intermediates if delete_intermediates=True. Saves resulting files as compressed .tsv files
+    using compression method given by compression. Does not recompute previously saved files.
+    """
+    for mp in (metapath.inverse, metapath):
+        combined_path = hetmat.get_summary_degree_group_path(metapath, 'dwpc', damping)
```

Comment:
Not ideal to use compression but not use compression extension.

We could create a compression argument for `get_summary_degree_group_path` that would get the extension using a dict like [here](https://github.com/pandas-dev/pandas/blob/6008d75ac00aa7c3052b74bf4f6eb73beeaf56a6/pandas/io/common.py#L253-L258)